.travis.yml

```diff
@@ -10,6 +10,7 @@ before_install:
   - jq --version
   - curl -sSL -o ~/bin/install-jdk.sh https://raw.githubusercontent.com/sormuras/bach/master/install-jdk.sh && chmod +x ~/bin/install-jdk.sh
   - source ./.travis/.travis_set_deploy_build_opts.sh
+  - npm install -g npm-login-noninteractive
```

Comment:
I think it'd be good to put this line in `.travis_set_deploy_build_opts.sh` if not all builds need this.